lib/api/core/hotelClerk.js

```diff
@@ -366,6 +395,9 @@ class HotelClerk {
 
         return this.kuzzle.pluginsManager.trigger('room:new', {index, collection, roomId: response.id})
           .then(modified => {
+            // pipes are not allowed to modified the room id
+            const roomId = response.id;
```

Comment:
should also replace the return statement (line 421):
` return {diff: response.diff, roomId: modified.roomId};`
=>
` return {diff: response.diff, roomId};`